wallet-tool.py

```diff
@@ -270,14 +270,16 @@ def cus_print(s):
 elif method == 'listwallets':
     # Fetch list of wallets
     possible_wallets = []
-    for (dirpath, dirnames, filenames) in os.walk('wallets'):
+    dirname = os.path.split(os.path.abspath(__file__))
```

Comment:
similar to above, use `join` instead of `split`:
```
dirname = os.path.dirname(os.path.abspath(__file__))
wallet_dir = os.path.join(dirname, 'wallets')
```